app/org/sagebionetworks/bridge/StudyConsentBootstrapper.java

```diff
@@ -37,6 +37,7 @@ public StudyConsentBootstrapper(StudyService studyService, StudyConsentDao study
             study.setMinAgeOfConsent(18);
             study.setResearcherRole("api_researcher");
             study.setConsentNotificationEmail("bridge-testing+consent@sagebridge.org");
+            study.setSupportEmail("bridge-testing+support@sagebase.org");
```

Comment:
Has this email been verified? I'm not seeing it in the SES console, nor did I see any verification emails in bridge-testing@